spec/lib/iiif_print/jobs/child_works_from_pdf_job_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'spec_helper'
+require 'misc_shared'
+
+RSpec.describe IiifPrint::Actors::IiifPrintUploadActor do
+  # TODO: add specs
```

Comment:
I think it'll be good to make this an issue just for tracking purposes

Reply:
there is already an issue for this